.dockerignore

```diff
@@ -1,4 +1,6 @@
 bundles
+integration-cli/bundles
```

Comment:
Just a thought: I think this directory should be removed and unified to the top-level bundle directory, but probably it should be another PR